apps/server/src/modules/tldraw/utils/utils.ts

```diff
@@ -0,0 +1,263 @@
+import { setInterval, clearInterval } from 'timers';
+
+const Y = require('yjs');
+const syncProtocol = require('y-protocols/dist/sync.cjs');
+const awarenessProtocol = require('y-protocols/dist/awareness.cjs');
+
+const encoding = require('lib0/dist/encoding.cjs');
+const decoding = require('lib0/dist/decoding.cjs');
+const map = require('lib0/dist/map.cjs');
+
+const debounce = require('lodash.debounce');
+
+const callbackHandler = require('./callback').callbackHandler;
+const isCallbackSet = require('./callback').isCallbackSet;
+
+const CALLBACK_DEBOUNCE_WAIT = 2000;
+const CALLBACK_DEBOUNCE_MAXWAIT = 10000;
```

Comment:
Please move all the configuration-related consts to the module configuration and please make it possible to load each one of them via environment variable (you should use _Configuration_ class from _@hpi-schul-cloud/commons_ for that).